File: Basic Coding Problems/02 If - Elses/028 Write a program to input any character and check whether it is alphabet, digit or special character/Java/check_character.java
import java.util.Scanner;

public class check_character {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter any character: ");
        char character = scanner.next().charAt(0);

        if (Character.isAlphabetic(character)) {
            System.out.println("The character is an alphabet.");
        } else if (Character.isDigit(character)) {
            System.out.println("The character is a digit.");
        } else {
            System.out.println("The character is a special character.");
        }
    }
}

File: Basic Coding Problems/02 If - Elses/028 Write a program to input any character and check whether it is alphabet, digit or special character/C++/check_character.cpp
#include <iostream>

using namespace std;

int main() {
    char character;

    cout << "Enter any character: ";
    cin >> character;

    if (isalpha(character)) {
        cout << "The character is an alphabet.";
    } else if (isdigit(character)) {
        cout << "The character is a digit.";
    } else {
        cout << "The character is a special character.";
    }

    return 0;
}